src/prommis/nanofiltration/diafiltration.py

```diff
@@ -615,7 +693,7 @@ def Li_recovery_constraint(m):
 
         @m.Constraint()
         def Li_purity_constraint(m):
-            return m.Li_purity >= 0.8
+            return m.Li_purity >= 0.5
```

Comment:
Why did this change?

Reply:
I was adjusting these values when I was debugging the scaling/solver convergence. These constraints are not currently active for this example, though a user could choose to change this if desired.

Reply:
Thanks! Could you please add tests to cover those exceptions? In the File Comparison, CodeCov flags uncovered lines.